native/core/src/models/rsvp.rs

```diff
@@ -142,7 +142,7 @@ impl ActerModel for Rsvp {
 
         let manager = {
             let model = store.get(&belongs_to).await?;
-            if !model.capabilities().contains(&Capability::Commentable) {
```

Comment:
Does this mean we were wrongfully checking before ? As ```Commentable``` and ```RSVPable``` has different semantics?

Reply:
yes, we did. but as only events are rsvp-able and they are also commentable it didn't really matter in practice.